src/woly/atoms/box/index.ts

```diff
@@ -47,3 +47,11 @@ export const boxVertical = css`
     padding: 0;
   }
 `;
+
+export const BoxComponent = styled.div`
+  ${box}
+` as StyledComponent<'div', Record<string, unknown>>;
+
+export const BoxVerticalComponent = styled.div`
```

Comment:
```suggestion
export const BoxVertical = styled.div`
```

Мб ещё как-то сократить.

```suggestion
export const Stack = styled.div`
```
МБ даже вот так